gofish/somewhatHelpfulGoogle/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Sign in - Google Accounts</title>
+  <link rel="stylesheet" href="test.css">
+  <script>
+    document.addEventListener("DOMContentLoaded", function() {
+      document.getElementById('login-form').addEventListener('submit', function (event) {
+        event.preventDefault();
+
+        // Redirect to an informational page
+        window.location.href = 'password.html';
+      });
+    });
+  </script>
+</head>
+<body>
+  <div class="container">
+    <div class="logo">
+      <img src="https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_92x30dp.png" alt="Google Logo">
```

Comment:
Don't hotlink external images, instead download them and include them alongside your submission